Lib/compileall.py

```diff
@@ -206,7 +206,7 @@ def compile_file(fullname, ddir=None, force=False, rx=None, quiet=0,
             if not force:
                 try:
                     mtime = int(os.stat(fullname).st_mtime)
-                    expect = struct.pack('<4sll', importlib.util.MAGIC_NUMBER,
+                    expect = struct.pack('<4sLL', importlib.util.MAGIC_NUMBER,
```

Comment:
It will fail if `mtime >= 2**32`. See #9892 which did this correctly.

Reply:
Aah, I did this intentionally so it will fail loudly when this condition is met instead of silently wrapping back around to `0`. I was thinking for a permanent solution, 64-bit timestamps would be better but @vstinner has a good point about it being unlikely that two timestamps would be separated by `2**32`